README.md

```diff
@@ -5,7 +5,7 @@
 </h1>
 <!-- markdownlint-enable MD033 -->
 
-![Release](https://img.shields.io/badge/Latest%20Release-v3.0.2-blue)
+![Release](https://img.shields.io/badge/Latest%20Release-v3.0.3-blue)
```

Comment:
This should be a 3.1.0 instead of a 3.0.3; we are bumping minor versions of the components and dropping support for older kubernetes versions, this is not a bug fix.

Note: I'll comment once here to not spam the PR, the version should be changed in several places.